src/Momento.Sdk/Exceptions/CancelledException.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Momento.Sdk.Exceptions;
 
+using System;
+
 public class CancelledException : SdkException
 {
     /// <summary>
     /// Operation was cancelled.
     /// </summary>
-    public CancelledException(string message) : base(MomentoErrorCode.CANCELLED_ERROR, message)
-    {
-    }
-    public CancelledException(string message, MomentoErrorTransportDetails transportDetails) : base(MomentoErrorCode.CANCELLED_ERROR, message, transportDetails)
+    public CancelledException(string message, MomentoErrorTransportDetails? transportDetails=null, Exception? e=null) : base(MomentoErrorCode.CANCELLED_ERROR, message, transportDetails)
     {
+        this.MessageWrapper = "The request was cancelled by the server; please contact Momento";
```

Comment:
in all the places where I wrote "Please contact Momento" I think the latest suggestion was to change it to something like: "for assistance, please contact us at support@momentohq.com".